transport-native-unix-common/pom.xml

```diff
@@ -30,6 +30,8 @@
   </description>
 
   <properties>
+    <javaModuleName>io.netty.channel.unix</javaModuleName>
```

Comment:
Unix common no? It is a shared command module, not a unix implementation.

Reply:
If this module is just an "internal" stuff for the rest of Netty, I agree that we don't have to configure an automatic module name for it. It will be used just as a classpath dependency, which is possible for automatic modules (but not for true modules).

The only issue is that the manifest generator is in the parent POM, and removing this line would create an empty module name in the manifest.